iati/home/templates/home/home_page.html

```diff
@@ -41,38 +41,38 @@ <h2 class="section__heading">Getting started</h2>
                     <div class="help">
                         <div class="help__header background-cover" style="background-image: url(/static/assets/img/help1.png)">
                             <div class="help__panel fill-wind">
-                                <h3 class="help__heading">Explore our data</h3>
+                                <h3 class="help__heading">Data Use</h3>
                             </div>
                         </div>
                         <div class="help__body">
-                            <p class="help__body-copy">Etiam ornare diam sit amet tellus cursus rhoncus. Nam pretium pretium orci, at ullamcorper felis pharetra vitae.</p>
-                            <a class="button">Explore our data</a>
+                            <p class="help__body-copy">All people and organisations can be using IATI data. Find out what data includes, how you can access it and why you should be using it.</p>
+                            <a class="button">Explore IATI data</a>
                         </div>
                     </div>
                 </div>
                 <div class="l-3up__col">
                     <div class="help">
                         <div class="help__header background-cover">
                             <div class="help__panel fill-wind">
-                                <h3 class="help__heading">Our publishing guide</h3>
+                                <h3 class="help__heading">Guidance</h3>
                             </div>
                         </div>
                         <div class="help__body">
-                            <p class="help__body-copy">Etiam ornare diam sit amet tellus cursus rhoncus. Nam pretium pretium orci, at ullamcorper felis pharetra vitae.</p>
-                            <a class="button">Get started</a>
+                            <p class="help__body-copy">If you have questions about how you publish to IATI, where to start or how to improve your data - check out our publishing guidance.</p>
+                            <a class="button" href="/guidance/">Get started</a>
```

Comment:
Most of these pages already have helper functions to find the slug via the default page name. This will be more important when we get into multiple languages. In this case, as long as you `{% load iati_tags %}` at the start you can replace this href with `{% default_page_url 'guidance_and_support' %}`